### ISSUE TITLE
State

### ISSUE THREAD

**Issue by simcha-c** [amzick/lets_go_eros#3]
**users slice**
- [ ] users will probably have some more information about them no? bio, where they're from, etc, etc. we should add that into the users slice. you might only get that info on the users page, so some will have extra info and others will have basic data.
- [ ] if a user has many images and were having a images slice, we should have an `imagesId` array on the user.
- [ ] what data type is the birthday? will it be a string? please update that.

**images slice**
- [ ] it should be the 4: { id: 4, etc.. }. Please fix that.

**other slices**
- [ ] seems like youre missing some info in your state. You would need a `questions` slice, you would also need an `answers` slice. maybe theres other info that im not remembering right now? 
Basically, think about all of the information that you would need to get from your database, and that should be in your slice.

Best,
Simcha

**Comment by amzick** [comment]
Thanks for the feedback - I guess I based my state just based off of the home feed, which is basically just a series of user thumbnails with a match percentage. Not sure how to add an image here, can show a screen grab tomorrowI should be front loading the current users questions so they're ready to go?

**Comment by simcha-c** [comment]
So that is actually a great point. We should be doing just the home feed of a series of user thumbnails. Later on, you can add a user show page with more information. As soon as we have that, we can add info to the user. For now, it's good to think about it as - what info do i need for the thumbnail, and put that into the user slice.
Maybe for the current user, we'll need some more info? not sure

**Comment by simcha-c** [comment]
Even if you do have the thumbnail info for each user, you would still need a `questions` slice. Maybe other slices for the matching algo? I guess we'll talk tomorrow on the implementation...
